lightning/tests/lightning_pd_leader_switch/run.sh

```diff
@@ -43,7 +45,7 @@ pd-server --join "https://$PD_ADDR" \
 killall tidb-server
 # wait for TiDB to exit to release file lock
 sleep 5
-start_tidb
+PD_ADDR="${PD_ADDR},${PD_ADDR}2,${PD_ADDR}3" start_tidb
```

Comment:
we focus on testing lightning's behaviour of discovering new PD members, not testing TiDB. So passing all PD addresses to TiDB

Reply:
I think we do not to add it

Reply:
The results of this test show that it can still pass the test https://github.com/pingcap/tidb/pull/55491
https://do.pingcap.net/jenkins/blue/organizations/jenkins/pingcap%2Ftidb%2Fpull_lightning_integration_test/detail/pull_lightning_integration_test/1115/pipeline

Reply:
OK, I'll remove it

Reply:
https://github.com/pingcap/tidb/pull/55498/commits/0e6beb4ec1424710de1eca54d9ca9a6e8e077860